src/global_legacy_module/4C_global_legacy_module_validmaterials.cpp

```diff
@@ -2415,18 +2438,52 @@ std::unordered_map<Core::Materials::MaterialType, Core::IO::InputSpec> Global::v
                         "conductivity tensor. Setting its value to 1 resembles a scalar "
                         "conductivity, 2 or 3 a diagonal conductivity and 4 or 9 the full "
                         "conductivity tensor in two and three dimensions respectively."}),
-            parameter<std::vector<double>>("CONDUCT",
-                {.description =
-                        "Vector of values representing the thermal conductivity tensor in a "
-                        "row-wise ordering, with the vector size given by CONDUCT_PARA_NUM.",
-                    .size = from_parameter<int>("CONDUCT_PARA_NUM")}),
+            deprecated_selection<Inpar::Mat::ActivationType>("CONDUCT",
+                {{"tensor", Inpar::Mat::ActivationType::tensor},
+                    {"map", Inpar::Mat::ActivationType::map}},
+                {.description = "type of field evaluation"}),
```

Comment:
This would be `parameter<Inpar::Mat::ActivationType>` without manual listing of the options since the strings match the enum names. I can see that I should probably throw in this case and force you to use the correct one right away ;)